packages/back-end/src/util/stats.ts

```diff
@@ -1,4 +1,4 @@
-import { jStat } from "jstat";
+import cdf from "@stdlib/stats/base/dists/chisquare/cdf";
```

Comment:
I think we should have the distribution name in the source code. Can you import `chisquare` instead and have it be `chisquare.cdf(...)`?

I also might even prefer the more verbose `random.base.normal()` for the data generating methods, but definitely for quantile and cdf methods we should have them prefixed by their distribution in the code.